great_expectations/render/renderer/other_section_renderer.py

```diff
@@ -236,7 +236,7 @@ def _render_warnings(cls, evrs, content_blocks):
     def _get_percentage_missing_cells_str(cls, evrs):
 
         columns = cls._get_column_list_from_evrs(evrs)
-        if not columns or len(columns) == 9:
```

Comment:
This is a facepalm-worthy typo. The bug shows up only on data assets with 9 columns!